### PR TITLE
Archive and Trash Management

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Features Added in this PR : 

- Archive Note
- Restore Archived Note
- Delete Note
- Access Notes in Trash
- Restore Trash Notes

Files to look for : 

- Pages : `src/Pages/Archive` and `src/Pages/Trash`
- Context : `src/Context/ArchiveContext`
- Services : `src/Services/ArchiveServices`

Here is the Deploy Preview : https://deploy-preview-4--eclectic-choux-d9c480.netlify.app/

### CODE REVIEW

**Review by sushil-mod — APPROVED** [review]
everything looks fine. just a suggestion for future show error in UI and use barrel export means use index js file for exporting  all files from a single file

File: src/Context/ArchiveContext/ArchiveContext.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import { createContext, useContext, useEffect, useState } from "react";
+import { useAuth } from "../Auth/auth-context";
+import { useNotes } from "../NotesContext/NotesContext";
+import { addToArchiveNoteService } from "../../Services/ArchiveServices/addToArchiveService";
+import { deleteFromArchiveService } from "../../Services/ArchiveServices/deleteFromArchiveService";
+import { getArchiveNotesService } from "../../Services/ArchiveServices/getArchiveNotesService";
+import { restoreArchivedNotesService } from "../../Services/ArchiveServices/restoreArchivedNotes";
```
Comment (t007rushi) [inline src/Context/ArchiveContext/ArchiveContext.js]: use barrel export here

**Review by t007rushi — APPROVED** [review]
Else Look for fixed naming keep ```camelCase``` makes more readable

File: src/Context/ArchiveContext/ArchiveContext.js (hunk 1 above)
Comment (atharvabhanage02) [reply src/Context/ArchiveContext/ArchiveContext.js]: Done

**Pull request merged**